model/project_vars.go

```diff
@@ -106,7 +106,7 @@ func FindMergedProjectVars(projectID string) (*ProjectVars, error) {
 // UpdateProjectVarsByValue searches all projects who have a variable set to the toReplace input parameter, and replaces all
 // matching project variables with the replacement input parameter. If dryRun is set to true, the update is not performed.
 // We return a list of keys that were replaced (or, the list of keys that would be replaced in the case that dryRun is true).
-func UpdateProjectVarsByValue(toReplace, replacement, username string, dryRun bool) (map[string][]string, error) {
+func UpdateProjectVarsByValue(toReplace, replacement, username string, dryRun, enabledOnly bool) (map[string][]string, error) {
```

Comment:
Since `enabledOnly` isn't foolproof, it could cause some confusion about whether it's a bug or intentional that it doesn't work in some cases. Can we mention that `enabledOnly` is only respected if it's updating branch project vars and not a repo? We're going to need to change project vars for the secrets management project anyways, so knowing the purpose of this new special field would be good.